frida_tools/tracer.py

```diff
@@ -68,14 +70,15 @@ def _initialize(self, parser, options, args):
             self._decorate = options.decorate
             self._output = None
             self._output_path = options.output
+            self._init_session_paths = options.init_session
```

Comment:
Come to think of it now, it would be good to actually read the files here. This is so the user gets quick feedback if a typo is made, instead of having to wait until we've attached to the target process. (We also do it early in our REPL if loading a user-defined script.)

Reply:
I need to examine this one carefully.  I'm committing everything else and will implement this one in parallel.